release-calendar/README.md

```diff
@@ -19,15 +19,16 @@ To build the client for production: `npn run client-build-prod`
 * Make sure to set `ENV.NODE_ENV` to `production`
 * This uses React's production build
 
-To build the server: `npm run server-build`
-To serve the server: `npm run server-serve`
-To build and serve the server: `npm run server`
+To generate test data: `npm run setup`  
```

Comment:
Not necessarily in this PR, but `run setup` is a bad name for `give me fake data` :speak_no_evil:

Reply:
Ok~~